core/src/main/python/prepare_model.py

```diff
@@ -84,7 +85,7 @@ def main():
         _outputdir = model_context.get_output_dir()
         model_files = model_context.get_model_file()
 
-        obj = ModelPreparer(model_files, model_context, _outputdir)
+        obj = ModelPre https://github.com/jwgish/weblogic-deploy-tooling/pull/new/WDT-644parer(model_files, model_context, _outputdir)
```

Comment:
typo?

Reply:
I don't know how the heck this got in here, and I can't figure out how to get rid of it.